<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectExchangerDialogForm</class>
 <widget class="QDialog" name="DirectExchangerDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direct Exchanger</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_16">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_15">
     <item row="0" column="0">
      <widget class="QFrame" name="frame_3">
       <property name="styleSheet">
        <string notr="true">QWidget { background: white }</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_12" columnstretch="0,0">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>9</number>
        </property>
        <item row="1" column="1">
         <widget class="QLabel" name="m_imgLabel">
          <property name="minimumSize">
           <size>
            <width>112</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="text">
           <string>Imagem</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="m_titleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Direct Exchanger</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QToolButton" name="m_inputPGISToolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="text">
               <string>PostGIS</string>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QToolButton" name="m_inputADOToolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="text">
               <string>Access</string>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QToolButton" name="m_inputSHPToolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="text">
               <string>File</string>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Parameters</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_18">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_17">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Input Layer</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QComboBox" name="m_inputLayerComboBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Output Data Source</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QComboBox" name="m_outputDataSourceComboBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QToolButton" name="m_dsToolButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="0">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLineEdit" name="m_dataSetLineEdit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QToolButton" name="m_dirToolButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Data Set Name</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="m_spatialIndexCheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Create Spatial Index</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Output</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QToolButton" name="m_outputPGISToolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="text">
               <string>PostGIS</string>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QToolButton" name="m_outputADOToolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="text">
               <string>Access</string>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QToolButton" name="m_outputSHPToolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>54</width>
                <height>54</height>
               </size>
              </property>
              <property name="text">
               <string>File</string>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="0" column="0">
          <widget class="QPushButton" name="m_helpPushButton">
           <property name="text">
            <string>Help</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="m_okPushButton">
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="m_cancelPushButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_inputPGISToolButton</tabstop>
  <tabstop>m_inputADOToolButton</tabstop>
  <tabstop>m_inputSHPToolButton</tabstop>
  <tabstop>m_inputLayerComboBox</tabstop>
  <tabstop>m_outputDataSourceComboBox</tabstop>
  <tabstop>m_dataSetLineEdit</tabstop>
  <tabstop>m_dirToolButton</tabstop>
  <tabstop>m_spatialIndexCheckBox</tabstop>
  <tabstop>m_outputPGISToolButton</tabstop>
  <tabstop>m_outputADOToolButton</tabstop>
  <tabstop>m_outputSHPToolButton</tabstop>
  <tabstop>m_okPushButton</tabstop>
  <tabstop>m_cancelPushButton</tabstop>
  <tabstop>m_helpPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>DirectExchangerDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
